types/msg.go

```diff
@@ -350,6 +353,18 @@ type WasmMsg struct {
 	ClearAdmin   *ClearAdminMsg   `json:"clear_admin,omitempty"`
 }
 
+// These are messages in the IBC lifecycle using the new Eureka approach. Only usable by IBC-enabled contracts
+type EurekaMsg struct {
+	SendPacket *EurekaSendPacketMsg `json:"execute,omitempty"`
+}
+
+// Sends an IBC packet with given payloads over the existing channel.
+type EurekaSendPacketMsg struct {
```

Comment:
Could you add a test for this that deserializes from json generated from the cosmwasm-std types?